gogs_client/interface.py

```diff
@@ -1,12 +1,12 @@
 import requests
 
 from gogs_client._implementation.http_utils import RelativeHttpRequestor, append_url
-from gogs_client.entities import GogsUser, GogsRepo
+from gogs_client.entities import GogsUser, GogsRepo, TokenInfo
 
 
 class GogsApi(object):
     """
-    A Gogs client, serving as a wrapper around the Gogs HTTP API.
+    A Gogs API entries, serving as a wrapper around the Gogs HTTP API.
```

Comment:
<small><s>Because this is not consistent client with preset access configuration and persistent session (keep-alive and so on). This is collection of unrelated API wrappers.</s></small>
Sorry. I've reverted this accidental change.